sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -246,8 +250,18 @@ def __init__(self):
            'should not be initialized in the constructor.+'
            "Attribute 'you_should_not_set_this_' was found.+"
            'in estimator estimator_name')
-
     assert_raises_regex(AssertionError, msg,
                         check_no_fit_attributes_set_in_init,
                         'estimator_name',
                         NonConformantEstimator)
+
+
+if __name__ == '__main__':
+    main_module = sys.modules['__main__']
```

Comment:
Add a comment: "This module is run as a script to check that we have no dependency on pytest for estimator checks." Otherwise it could be removed by the ignorant and the test would still pass!